src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started digging a trench");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator has completed its digging task");
+    }
+
```

Comment:
```suggestion
```
fix the same issue in all places

Reply:
Build failure is without a new line at end of file.

Reply:
It means blank line at the end of file not after method. Please remove empty lines in all classes after methods